asyncwrapper/src/main/kotlin/org/n/riesgos/asyncwrapper/process/wps/WPSClientService.kt

```diff
@@ -10,7 +10,8 @@ class WPSClientService(val config : WPSConfiguration) {
 
     @Synchronized
     fun establishWPSConnection() : WPSClientSession{
-        val wpsClient = WPSClientSession.getInstance()
+        //val wpsClient = WPSClientSession.getInstance()
+        val wpsClient = WPSClientSession()
```

Comment:
Does that mean that `WPSClientSession.getInstance()` had returned a stateful singleton which would always return the same result for all your retries (from the previous commit)? Great thing that that is changed now.